recover-id-object/.dockerignore

```diff
@@ -0,0 +1,8 @@
+# Assumes docker context is repository root, i.e. ../../ relative to this file.
+*
+!deps/concordium-rust-sdk
+deps/concordium-rust-sdk/target
+!recover-id-object/*
+recover-id-object/target
+recover-id-object/scripts
+recover-id-object/*.md
```

Comment:
Please don't ignore this, especially the md files. Sometimes these are embedded in the binary and builds fail.

These are light files, no reason to complicate maintenance by ignoring htem.